tests_e2e/test_suites/images.yml

```diff
@@ -101,7 +101,6 @@ images:
    mariner_2_arm64:
       urn: "microsoftcblmariner cbl-mariner cbl-mariner-2-arm64 latest"
       locations:
-         AzureCloud: ["eastus"]
```

Comment:
Image is available in other locations too

Reply:
I don't know the reason why you restricted. My guess is at that time image may not be available but now it's available in canary too.